views/includes/inputs/date/date-of-birth-input.html

```diff
@@ -1,5 +1,58 @@
+{% set dobErrorMessage = '' %}
+{% set isDobDayError = false %}
+{% set isDobMonthError = false %}
+{% set isDobYearError = false %}
+
+{% if errors %}
```

Comment:
Maybe just add some comments to this code added to the templates - which bits are doing what? I think some bits are highlighting the fields in red, another bit is placing focus in the field etc.

Reply:
updated